test-lib-openshift.sh

```diff
@@ -460,7 +494,19 @@ function ct_os_test_s2i_app_func() {
 
   ct_os_new_project
   # Create a specific imagestream tag for the image so that oc cannot use anything else
-  ct_os_upload_image "${image_name}" "${image_tagged}"
+  if [ "${CT_SKIP_UPLOAD_IMAGE:-false}" == 'true' ] ; then
```

Comment:
Do we have usage for both cases? Wouldn't `oc import-image` fits our current usage in tests too? @pkubatrh Ideas?

Reply:
I'm not sure whether import-image works with the local docker -- if it would, then it would make it easier. @pvalena might know maybe, have you tried that?

Reply:
AFAIR It should work, if you already have the image locally. But then you have to setup the connection to `remote registry` in your OpenShift instance and use it for the import.

Reply:
Looking at it more `--from=docker.io/openshift/ruby-20-centos7` should probably work even with the IP of the `host`. But I tried the access to host (via some IP), so it may be blocked internally by some OpenShift routing system.

Reply:
But if you only have the image in the local registry, that is not visible to the remote cluster, this won't work.